package.json

```diff
@@ -1,121 +1,119 @@
-        "cz-conventional-changelog": "^3.2.0",
-        "eslint": "^6.8.0",
-        "eslint-config-airbnb-base": "5.0.1",
-        "eslint-config-prettier": "^6.11.0",
-        "eslint-config-salesforce": "^0.1.0",
-        "eslint-config-salesforce-license": "^0.1.0",
-        "eslint-config-salesforce-typescript": "^0.1.0",
-        "eslint-plugin-header": "^3.0.0",
-        "eslint-plugin-import": "^2.20.2",
-        "eslint-plugin-jsdoc": "^27.0.3",
-        "eslint-plugin-prettier": "^3.1.3",
-        "husky": "^4.2.5",
-        "mocha": "^7.2.0",
-        "nyc": "^15.1.0",
-        "prettier": "^2.0.5",
-        "pretty-quick": "^2.0.1",
-        "sinon": "^9.0.2",
-        "ts-node": "^8.10.2",
-        "typescript": "^3.9.3"
-    },
-    "config": {
-        "commitizen": {
-            "path": "cz-conventional-changelog"
-        }
-    },
-    "engines": {
-        "node": ">=12.0.0"
-    },
-    "homepage": "https://github.com/salesforcecli/salesforcedx",
-    "keywords": [
-        "force",
-        "salesforce",
-        "sfdx",
-        "salesforcedx",
-        "sfdx-plugin"
-    ],
-    "license": "BSD-3-Clause",
-    "repository": "salesforcecli/salesforcedx",
-    "scripts": {
-        "build": "sf-build",
-        "clean": "sf-clean",
-        "clean-all": "sf-clean all",
-        "compile": "sf-compile",
-        "docs": "sf-docs",
-        "format": "sf-format",
-        "lint": "eslint 'src/**/*.ts'",
-        "prepack": "sf-build",
-        "test": "echo 'Salesforcedx is an aggregator plugin that does not contain any tests'",
-        "version": "oclif-dev readme && git add README.md"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "sf-husky-pre-commit",
-            "commit-msg": "sf-husky-commit-msg",
-            "pre-push": "sf-husky-pre-push"
-        }
-    },
-    "publishConfig": {
-        "access": "public"
+  "name": "salesforcedx",
+  "version": "50.14.4",
+  "description": "The Salesforce DX plugin aggregates all force topic commands into a single plugin that is bundled with the Salesforce CLI.",
+  "author": "Salesforce",
+  "bugs": "https://github.com/forcedotcom/cli/issues",
+  "files": [
+    "/lib",
+    "/messages",
+    "/oclif.manifest.json"
+  ],
+  "oclif": {
+    "commands": "./lib/commands",
+    "plugins": [
+      "salesforce-alm",
+      "@salesforce/plugin-templates",
+      "@salesforce/sfdx-plugin-lwc-test",
+      "@salesforce/plugin-custom-metadata",
+      "@salesforce/plugin-apex",
+      "@salesforce/plugin-schema",
+      "@salesforce/plugin-user",
+      "@salesforce/plugin-limits"
+    ]
+  },
+  "pinnedDependencies": [
+    "@salesforce/plugin-alias",
+    "@salesforce/plugin-custom-metadata",
+    "@salesforce/plugin-limits",
+    "@salesforce/plugin-schema",
+    "@salesforce/plugin-templates",
+    "@salesforce/plugin-user",
+    "@salesforce/sfdx-plugin-lwc-test",
+    "salesforce-alm"
+  ],
+  "dependencies": {
+    "@salesforce/plugin-alias": "1.1.5",
```

Comment:
did you mean to add alias? I thought it lived at the cli level?

Reply:
We decided not to top level alias, so it still lives in force. While it lives in force, I think it should be under salesforcedx. We can remove it from sfdx-cli at another time, it doesn't have to align perfectly.